test/check-application

```diff
@@ -73,6 +74,19 @@ class TestNavigator(testlib.MachineCase):
         b.click(f"#create-link-{filetype}")
         b.click("button.pf-m-primary")
 
+    def allowDownload(self):
+        b = self.browser
+        if b.cdp.browser.name == "chromium":
+            b.cdp.invoke("Page.setDownloadBehavior", behavior="allow", downloadPath=b.cdp.download_dir)
+
+    def waitDownloadFile(self, filename):
+        b = self.browser
+        def download_file():
+            return glob.glob(os.path.join(b.cdp.download_dir, filename))
+
+        # chromium saves as __newfile__
+        testlib.wait(lambda: len(download_file()) > 0)
```

Comment:
This only checks if it started the downloading, not that it completed. Presumably needs to get passed an expected size or content?

Reply:
We could try testing cancelDownload as well:
https://chromedevtools.github.io/devtools-protocol/tot/Browser/#method-cancelDownload

Not sure how we check if the `fsread` channel is closed however.

For the big file use `truncate -s 1G`

Reply:
Added a test to check for expected size, content and one big download.